src/librustc_borrowck/diagnostics.rs

```diff
@@ -314,6 +314,72 @@ let c = &i; // still ok!
 ```
 "##,
 
+E0501: r##"
+This error indicates that a mutable variable which has been captured in a
+closure is being used while it is still captured by the closure. Because the
+closure has borrowed the variable, the variable is not available for use until
```

Comment:
While I agree that using one term would be helpful, to switch everything to "borrow" would be misleading.  [Captures](http://rustbyexample.com/fn/closures/capture.html) are a separate concept than borrows. I think that linking to an explanation of a capture may be helpful, since a capture may move or borrow an item.  What are your thoughts?